lib/tests/Utils.hpp

```diff
@@ -11,6 +11,7 @@
 
 #ifdef __APPLE__
 #   include <stdexcept>
+#   include <unistd.h> // For mkdtemp
```

Comment:
Shouldn't that be
1. `cstdlib` and
2. unconditional, as this is the official header for the declaration of `mkdtemp`?